.rubocop.yml

```diff
@@ -6,8 +6,7 @@ AllCops:
   Exclude:
     - 'apps/**/*'
     - 'code-studio/**/*'
-    - 'dashboard/db/**/*'
-    - 'dashboard/db/*'
+    - 'dashboard/db/schema.rb'
```

Comment:
Note that this file is auto-generated by rails, so blacklisting the offending individual lines seems cumbersome.